.zuul.yaml

```diff
@@ -212,6 +212,18 @@
             override-checkout: main
             required-projects:
               - github.com/openstack-k8s-operators/watcher-operator
+            irrelevant-files: &irrelevant_files_watcher
+            # From watcher irrelevant_files from watcher-grenade job job: https://github.com/openstack/watcher/blob/master/.zuul.yaml#L151
+              - ^(test-|)requirements.txt$
```

Comment:
I'm +1 on this change overall, the only one I have doubts about is the `requirements.txt` file for the `openstack/watcher` repo. Do we use that file when building the containers?

Reply:
I have commented out in new patchset.

Reply:
ack, thanks for explaining, lgtm now